jobQueue.py

```diff
@@ -313,7 +315,7 @@ def makeDead(self, id, reason):
         self.log.debug("makeDead| Released lock to job queue.")
         return status
 
-    def getInfo(self):
+    def getInfo(self) -> dict:
```

Comment:
`Dict` should be capitalized and contain the type mapping, see https://docs.python.org/3/library/typing.html#typing.Dict